app/jobs/index_canvas_job.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+##
+# Used to kick off the CanvasResource
+class IndexCanvasJob < ActiveJob::Base
```

Comment:
I'm not sure I'm understanding why this job was extracted from `IiifCanvasIndexerEnqueuer`?